src/App.tsx

```diff
@@ -1,28 +1,112 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+
+const USER_ID = 11926;
+
+enum FilterType {
```

Comment:
```suggestion
enum Filter {
```
we already define enum type - no need to duplicate it in the name